### DISCUSSION TITLE
How to generate access logs?

### DISCUSSION THREAD

**Discussion by proofy** [static-web-server/static-web-server#206] [Q&A]
I want to analyse the web traffic, so I need access logs with all the data from the clients.
How to enable/change the configuration to generate access logs?

**Comment by joseluisq** [comment]
Usually enabling the `info` log level. 

```sh
static-web-server --port 8787 --root ./my-public-dir --log-level "info"
```

In addition, if you want to log remote addresses or real remote IPs then you can use `--log-remote-address=true` option.
You can check the docs for more details https://static-web-server.net/features/logging/

**Comment by proofy** [comment]
In comparison with the apache logs I'm missing 
- status
- size in byte
- referer  
- user-agent
- host

**Reply by joseluisq** [reply]
I see, you want something like Apache's [LogFormat](https://httpd.apache.org/docs/2.4/mod/mod_log_config.html#logformat). Unfortunately, this is not supported.
However, we could implement it as a new feature. So feel free to open a feature request.

**Reply by proofy** [reply]
Thank you, it's just good to know, that this is not possible. So it could be a good idea to use it together with traefik. I have to make some test also with virtual hosts etc.
